testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -7869,12 +7869,16 @@ private static Consumer<List<? extends Throwable>> checkWarnings(
   @Test void testNamedParameterOperators() {
     expr("@q")
         .ok("@q");
-    expr("a > @Q")
-        .ok("(`A` > @Q)");
-    expr("a + @Q")
-        .ok("(`A` + @Q)");
-    expr("1 <=> @WeWon")
-        .ok("(1 <=> @WeWon)");
+    expr("$Z")
+        .ok("$Z");
+//    expr("a > @Q")
```

Comment:
What do these tests do and why are they commented out?

Reply:
Fixed.